### PR TITLE
Add empty definitions for did core terms

### PR DESCRIPTION
**Pull request by OR13**
Blocked by: https://github.com/perma-id/w3id.org/pull/1638

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1232,7 +1232,183 @@ <h3>signatureAlgorithm</h3>
 }
         </pre>
       </section>
+
+      <section id="EcdsaSecp256k1Signature2019" about="https://w3id.org/security#EcdsaSecp256k1Signature2019"
+      typeof="rdf:Property">
+      <h3>EcdsaSecp256k1Signature2019</h3>
```
Comment (davidlehn) [inline index.html]: All missing a 2 space indent.

File: index.html [hunk 2]
```diff
@@ -1232,7 +1232,183 @@ <h3>signatureAlgorithm</h3>
 }
         </pre>
       </section>
+
+      <section id="EcdsaSecp256k1Signature2019" about="https://w3id.org/security#EcdsaSecp256k1Signature2019"
```
Comment (davidlehn) [inline index.html]: Might be better if these were sorted with the above items rather than all at the bottom.

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html]: indented

File: index.html (hunk 2 above)
Comment (OR13) [reply index.html]: I think its probably better to keep them grouped at the bottom until they are defined... that way when we define them, we can move them to the right place as we go.

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html]: can i resolve this?

Comment (davidlehn) [reply index.html]: Yes. I don't have access to.

File: index.html [hunk 3]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
 }
         </pre>
       </section>
-    </section>
 
+      <section id="EcdsaSecp256k1Signature2019" about="https://w3id.org/security#EcdsaSecp256k1Signature2019"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 4]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>EcdsaSecp256k1Signature2019</h3>
+      </section>  
+
+      <section id="EcdsaSecp256k1VerificationKey2019" about="https://w3id.org/security#EcdsaSecp256k1VerificationKey2019"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 5]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>EcdsaSecp256k1Signature2019</h3>
+      </section>  
+
+      <section id="Ed25519Signature2018" about="https://w3id.org/security#Ed25519Signature2018"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 6]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>Ed25519Signature2018</h3>
+      </section>  
+
+      <section id="Ed25519VerificationKey2018" about="https://w3id.org/security#Ed25519VerificationKey2018"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 7]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>Ed25519VerificationKey2018</h3>
+      </section>  
+
+      <section id="RsaSignature2018" about="https://w3id.org/security#RsaSignature2018"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 8]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>RsaSignature2018</h3>
+      </section>  
+
+      <section id="RsaVerificationKey2018" about="https://w3id.org/security#RsaVerificationKey2018"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 9]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>RsaVerificationKey2018</h3>
+      </section> 
+
+      <section id="SchnorrSecp256k1Signature2019" about="https://w3id.org/security#SchnorrSecp256k1Signature2019"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 10]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>SchnorrSecp256k1Signature2019</h3>
+      </section> 
+
+      <section id="SchnorrSecp256k1VerificationKey2019" about="https://w3id.org/security#SchnorrSecp256k1VerificationKey2019"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

File: index.html [hunk 11]
```diff
@@ -1232,7 +1232,176 @@ <h3>signatureAlgorithm</h3>
...
+        <h3>SchnorrSecp256k1VerificationKey2019</h3>
+      </section> 
+
+      <section id="ServiceEndpointProxyService" about="https://w3id.org/security#ServiceEndpointProxyService"
+      typeof="rdf:Property">
```
Comment (dlongley) [inline index.html]:
```suggestion
      typeof="rdfs:Class">
```

**Comment by OR13** [comment]
@msporny @dlongley @dmitrizagidulin @davidlehn can I get some reviews / approvals on this? as a CCG admin, I can merge without approval, but I don't ever like doing that. If we don't want me to have that power here, we should apply branch protection and not exempt admins.

**Pull request merged**